.github/workflows/tests.yml

```diff
@@ -83,22 +83,21 @@ jobs:
 
             if (label_names.includes('CI-syntax-only')) {
               console.log('CI-syntax-only label found. Skipping tests job.')
-              core.setOutput('syntax-only', 'true')
               syntax_only = true
             } else {
               console.log('No CI-syntax-only label found. Running tests job.')
-              core.setOutput('syntax-only', 'false')
             }
 
             if (label_names.includes('CI-published-bottle-commits')) {
               console.log('CI-published-bottle-commits label found. Skipping tests job.')
-              core.setOutput('syntax-only', 'true')
               syntax_only = true
             }
 
             if (syntax_only) {
+              core.setOutput('syntax-only', 'true')
```

Comment:
I wonder if we can even just get away with one call that goes
```javascript
core.setOutput('syntax-only', syntax_only)
```

Reply:
Thanks, [that's true](https://github.com/actions/toolkit/blob/main/packages/core/src/core.ts#L185-L192):

```typescript
/**
 * Sets the value of an output.
 *
 * @param     name     name of the output to set
 * @param     value    value to store. Non-string values will be converted to a string via JSON.stringify
 */
// eslint-disable-next-line @typescript-eslint/no-explicit-any
export function setOutput(name: string, value: any): void {
```

```console
$ node
Welcome to Node.js v18.15.0.
Type ".help" for more information.
> JSON.stringify(true)
'true'
```